__docs__/wonder-blocks-tokens/box-shadow.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import {Title, Subtitle, Description, Stories} from "@storybook/blocks";
+import {Meta} from "@storybook/react";
+import {View} from "@khanacademy/wonder-blocks-core";
+import TokenTable from "../components/token-table";
+import {
+    boxShadow,
+    semanticColor,
+    sizing,
+} from "@khanacademy/wonder-blocks-tokens";
+import ComponentInfo from "../components/component-info";
+import packageConfig from "../../packages/wonder-blocks-tokens/package.json";
+import {Code} from "../components/code";
+
+/**
+ * The `boxS` tokens are used to define the box-shadow of an element. This
+ * includes both the elevation and the shadow color of it.
+ *
+ * ## Usage
+ *
+ * ```ts
+ *   import {boxShadow} from "@khanacademy/wonder-blocks-tokens";
+ *   const styles = {
+ *       elevatedContainer: {
+ *           boxShadow: boxShadow.low,
+ *       },
+ *   };
+ * ```
+ */
+export default {
+    title: "Packages /Tokens / boxShadow",
+    parameters: {
+        docs: {
+            // Use a custom page so the SB <Primary> component is not rendered
+            // <Primary> renders the first story as part of the description,
+            // which isn't necessary for the token pages.
+            page: () => (
+                <>
+                    <Title />
+                    <Subtitle />
+                    <Description />
+                    <Stories title="Tokens" />
+                </>
+            ),
+        },
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+    },
+    tags: ["!dev"],
+} as Meta;
+
+type Row = {label: string; css: string; value: string};
+
+export const Elevation = {
```

Comment:
Should the title of the story be `boxShadow` since it's the box shadow tokens?

<img width="1191" height="846" alt="Image" src="https://github.com/user-attachments/assets/9a25d49f-7a5f-4ea3-be9e-85baf9d41e30" />